.rubocop.yml

```diff
@@ -1,6 +1,5 @@
 inherit_from:
   - ./config/default.yml
-  - ./config/rails.yml
```

Comment:
So running RuboCop as CI for the repo has two purposes:
- lints `Gemfile`,  `*.gemspec` and the `Rakefile` in the repo. 
- provides a rudimentary check that RuboCop can parse those files and the config is valid. Essentially all we do to test the config.

So `./config/rails.yml` is just included to make sure it's valid.

Reply:
Ah, I see. I've moved this to a separate PR: https://github.com/alphagov/rubocop-govuk/pull/40.